[2022-11-22 05:29:27,744] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.load_files.extract_weather_info manual__2022-11-22T05:29:06.698630+00:00 [queued]>
[2022-11-22 05:29:27,801] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.load_files.extract_weather_info manual__2022-11-22T05:29:06.698630+00:00 [queued]>
[2022-11-22 05:29:27,802] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 05:29:27,803] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-22 05:29:27,804] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 05:29:27,887] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_files.extract_weather_info> on 2022-11-22 05:29:06.698630+00:00
[2022-11-22 05:29:27,903] {standard_task_runner.py:52} INFO - Started process 555 to run task
[2022-11-22 05:29:27,919] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'load_files.extract_weather_info', 'manual__2022-11-22T05:29:06.698630+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmpqt78y_ua', '--error-file', '/tmp/tmpe2vj0zjf']
[2022-11-22 05:29:27,923] {standard_task_runner.py:80} INFO - Job 259: Subtask load_files.extract_weather_info
[2022-11-22 05:29:28,200] {task_command.py:369} INFO - Running <TaskInstance: etl_solar.load_files.extract_weather_info manual__2022-11-22T05:29:06.698630+00:00 [running]> on host 8a647070b091
[2022-11-22 05:29:28,616] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=load_files.extract_weather_info
AIRFLOW_CTX_EXECUTION_DATE=2022-11-22T05:29:06.698630+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-22T05:29:06.698630+00:00
[2022-11-22 05:29:28,658] {base.py:68} INFO - Using connection ID 'mysql_solar' for task execution.
[2022-11-22 05:29:28,840] {python.py:173} INFO - Done. Returned value was: None
[2022-11-22 05:29:28,919] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=load_files.extract_weather_info, execution_date=20221122T052906, start_date=20221122T052927, end_date=20221122T052928
[2022-11-22 05:29:29,021] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-22 05:29:29,304] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
